package.json

```diff
@@ -7,14 +7,18 @@
     "start": "nodemon",
     "deploycommands": "npx ts-node ./src/deploy.commands.ts",
     "deletecommands": "npx ts-node ./src/delete.commands.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "auth": "ts-node ./src/auth.ts"
```

Comment:
Is this command supposed to be run in a separate terminal along with the bot?

If that is the case, then can you make it so that the Express server starts automatically before the bot starts? You can export the Express app and then call the listen method in `index.ts`

Reply:
okay